### DISCUSSION TITLE
Group Similar or Consecutive Traces

### DISCUSSION THREAD

**Discussion by rvndbalaji** [langfuse/langfuse#3361] [Ideas]
### Describe the feature or potential improvement

I have an application that are used by both human users as well as for internal application where an API sends a bulk LLM request

Problem :
 - Bulk requests are taking up most of the traces
 - Difficult for me to find an overview of all traces in the last X time (since the bulk traces take up all the space)

* Provide a way to group successive requests by user 
* This allows a neat organization of all traces and the group can be expanded to see the bulk request

**Comment by marcklingen** [comment]
Thanks for sharing. I’d suggest to use:

- userId to associate traces with users
- trace.name or tags to differentiate between use cases, eg api vs end-user requests
- sessionId to group subsequent requests by a user
